crates/project/src/project_tests.rs

```diff
@@ -90,6 +90,127 @@ async fn test_symlinks(cx: &mut gpui::TestAppContext) {
     });
 }
 
+#[gpui::test]
+async fn test_editorconfig_support(cx: &mut gpui::TestAppContext) {
+    init_test(cx);
+    cx.executor().allow_parking();
+
+    let dir = temp_tree(json!({
+        ".editorconfig": r#"
+			root = true
+			[*.rs]
+			indent_style = tab
+			indent_size = 3
+			end_of_line = lf
+			insert_final_newline = true
+			trim_trailing_whitespace = true
+			max_line_length = 80
+		"#,
+        ".zed": {
+            "settings.json": r#"{
+                "tab_size": 8,
+                "hard_tabs": false,
+                "ensure_final_newline_on_save": false,
+                "remove_trailing_whitespace_on_save": false,
+                "preferred_line_length": 64,
+                "soft_wrap": "none"
+            }"#,
+        },
+        "a.rs": "fn a() {\n    A\n}",
+        "b": {
+            ".editorconfig": r#"
+				[*.rs]
+				indent_size = 2
+			"#,
+            "b.rs": "fn b() {\n    B\n}",
+        },
+        "README.md": "tabs are better\n",
+    }));
+
+    let path = dir.path();
+    let fs = FakeFs::new(cx.executor());
+    fs.insert_tree_from_real_fs(path, path).await;
+    let project = Project::test(fs, [path], cx).await;
```

Comment:
I wanted to use:
```suggestion
    let project = Project::test(Arc::new(RealFs::default()), [path], cx).await;
```

But somehow the `.zed/settings` file is not detected that way... Anyone knows what I am missing ?